data_policy.md

```diff
@@ -1,17 +1,21 @@
 # ISIS Data Policy  
 
-## ISIS data management policy
+The ISIS data policy lays out the guarantees ISIS provides, and the requirements for users of the facilities, pertaining to the curation, access, ownership, use and storage of data collected at ISIS.
+
+The policy applies to anyone using ISIS facilities to produce experimental data, except as part of 'commercial-in-confidence' access to ISIS. Commercial users must instead come to an agreement about how their data is to be handled with ISIS before running their experiment.
+
+ISIS is committed to the principles of FAIR data in order to support research, and the advancement of knowledge and technology. We release data in common, easily accessible formats under a permissive Creative Commons license, along with detailed metadata in a variety of human and machine readable formats.
```

Comment:
```suggestion
ISIS is committed to support research and the advancement of knowledge and technology. We release data in easily accessible formats under a permissive Creative Commons license, along with metadata in a variety of human and machine readable formats.
```

Reply:
I have removed the 'principles of FAIR' simply because if we mention FAIR, which not all our users know what is, we need to also define it and reference it. As needed Elizabeth, Abigail etc can advice on worded on this when we send these suggestions for changes round

Reply:
The rest of the line was only really there to support the 'principles for FAIR data' bit - I don't think it makes sense to leave it in without that. I'd suggest we either remove this line entirely, or make `FAIR` a link to an explanation (maybe https://www.go-fair.org/fair-principles/ or the wikipedia page?), as I agree that it's not good to just drop it in without explanation.